config/routes.rb

```diff
@@ -8,6 +8,12 @@
       resources :products , only: [:index] do
         resources :product_versions, only: [:index]
       end
+      resources :product_versions,  only: [:index] do
+        resources :screenshots,  only: [:index, :new, :create]
+      end
+      resources :locales,  only: [:index] do
+        resources :screenshots,  only: [:index, :new, :create]
+      end
```

Comment:
I thought we were more on the lines of implementing it in this way: `product/product_versions/screenshots`. Correct me if I am wrong here.